python_modules/dagster/dagster/_core/definitions/freshness_checks/last_update.py

```diff
@@ -28,7 +28,7 @@
 
 
 @experimental
-def build_freshness_checks_for_non_partitioned_assets(
+def build_last_update_freshness_checks(
```

Comment:
That each of the variants calls an identically named function called `_build_freshness_checks_for_asset` the likve in different was actually quite confusing. It made it appeared we were exposing two different heavy APIs for light specialization of the same underlying logic when superficially examined, rather than completely separate business logic.